boards/nxp/imx93_evk/Kconfig

```diff
@@ -0,0 +1,16 @@
+# i.MX 93 EVK board configuration
+
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_MIMX93_EVK_EXP_SEL_INIT
+	bool "Configure i.MX 93 EVK board mux control during init"
+	default n
```

Comment:
```
default y
depends on GPIO_ADP5585
```
So that if GPIO ADP5585 is not enabled, this item will not be enabled.

Reply:
Current implimentation is the other way around:

if BOARD_MIMX93_EVK_EXP_SEL_INIT then enable GPIO_ADP5585. See Kconfig.defconfig .  Should we change this?